code/scripting/api/objs/object.cpp

```diff
@@ -194,6 +216,23 @@ ADE_VIRTVAR(Orientation, l_Object, "orientation", "Object world orientation (Wor
 	return ade_set_args(L, "o", l_Matrix.Set(matrix_h(&objh->objp->orient)));
 }
 
+ADE_FUNC(SetOrientNoCollideRetime, l_Object, "orientation", "Object world orientation (World orientation)", "vector", "Updates the orientation WITHOUT recalculating collisions. This is faster, but may cause incorrect collisions if moved too far.")
```

Comment:
Much as I'm reluctant to make this function name even more verbose, I think `Orientation` should be spelled out, since it's spelled out in the property.